x-pack/plugins/apm/public/services/filters/get_encoded_es_filters.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { ESFilter } from 'elasticsearch';
+import { UIFilters } from '../../../typings/ui-filters';
+import { getEnvironmentESFilter } from './get_environment_es_filter';
+import { getKueryESFilter } from './get_kuery_es_filter';
+
+export async function getEncodedEsFilters(
+  uiFilters: UIFilters
+): Promise<string> {
+  const kuery = await getKueryESFilter(uiFilters.kuery);
+  const environment = getEnvironmentESFilter(uiFilters.environment);
+
+  const filters = [kuery, environment].filter(filter => !!filter) as ESFilter[];
```

Comment:
Why is it useful to cast it to `ESFilter[]`?

Reply:
it was left over from a refactor sweep i did. thanks for catching this.